clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -589,12 +592,20 @@ public static Set<TopicPartition> generateRandomTopicPartitions(int numTopic, in
      * @return The caught exception cause
      */
     public static <T extends Throwable> T assertFutureThrows(Future<?> future, Class<T> exceptionCauseClass) {
```

Comment:
We introduced the timeout to this helper method, so I said we should also update the Javadoc.